### DISCUSSION TITLE
Was the UW code base affected by the Log4J Vulnerability?

### DISCUSSION THREAD

**Discussion by shahahmad** [universalweb/Network#10] [Q&A]
Out of curiosity, was the UW code base affected at all by the Log4J vulnerability? I noticed one of the more recent commits cleaned up and updated a ton of the packages being pulled in.

**Comment by tomekmarchi** [comment]
Nope "Log4j is a Java-based logging utility" "Java" - https://github.com/universalweb/Network/search?q=Log4j - This is all Node
